spec/controllers/concerns/client_sortable_spec.rb

```diff
@@ -117,7 +117,7 @@ def filter_cookie_name
 
       it "creates a query for the search and scopes to vita partner" do
         expect(subject.filtered_and_sorted_clients).to eq clients_query_double
-        expect(clients_query_double).to have_received(:where).with('vita_partners.id IN (:id)', id: [vita_partner.id])
+        expect(clients_query_double).to have_received(:where).with({:vita_partner_id=>[vita_partner.id]})
```

Comment:
Syntax suggestion: You can use `.with(vita_partner_id: [vita_partner.id])` I believe.

Not a blocker. If you make that change, feel free to push.